spec/models/adventure_spec.rb

```diff
@@ -3,27 +3,19 @@
 describe Adventure do
   subject { FactoryGirl.build(:adventure) }
 
-  it { should have_many(:bucket_list_adventures) }
+  it { should have_many(:bucket_list_adventures).dependent(:destroy) }
+  it { should have_many(:users) }
 
-  # it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
-  # it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
-  # it { should_not have_valid(:name).when(nil, '') }
-  # it { should_not have_valid(:address).when(nil, '') }
+  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
+  it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.